pkgs/applications/version-management/git-lfs/default.nix

```diff
@@ -19,7 +19,8 @@ buildGoPackage rec {
 
   subPackages = [ "." ];
 
-  preBuild = ''
+  # fork/exec /build/go-build2056524003/b001/exe/mangen: exec format error
+  preBuild = lib.optionalString (stdenv.hostPlatform == stdenv.buildPlatform)''
```

Comment:
This will most likely lead to a compile error because the generated file is not in git.. Please mark it broken instead.

Reply:
made postBuild and postInstall conditional